src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double base1;
+    private double base2;
+    private double height;
+
+    public IsoscelesTrapezoid(String color, double base1, double base2, double height) {
+        super(color);
+        this.base1 = base1;
+        this.base2 = base2;
+        this.height = height;
+    }
+
+
+    @Override
+    public double area() {
```

Comment:
The method name `area` should be a noun, as it represents a property of the object. Consider renaming it to something like `getArea` to follow Java naming conventions.